pnpm-workspace.yaml

```
@@ -0,0 +1,5 @@
+packages:
+  - '.'
+  - 'web'
+  # Root directory for shared scripts and dependencies
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)